.github/workflows/test.yml

```diff
@@ -23,17 +19,39 @@ on:
 
 jobs:
   ci:
-    runs-on: <image-name>
+    runs-on: ubuntu-20.04
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
-      - name: Use <setup tooling>
-        uses: <action to setup tooling>
+      - name: Setup Odin
+        uses: laytan/setup-odin@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+          # TODO: Pull a specific version of Odin compatible with the OS.
+          # The current setting pulls the latest version of Odin from the repo, and rebuilds it from
+          # scratch. This is slow, and clutters the action's log. Rather ensure that the OS has the
+          # right dependencies for a recent versioned release of the compiler.
+          release: false
+
+          llvm-version: 14
 
       - name: Install project dependencies
-        run: <install dependencies>
+        run: bin/fetch-configlet
```

Comment:
https://github.com/exercism/github-actions can fetch configlet and lint the track for you so we don’t need to roll our own action to do that. That’s one less thing to worry about going forward.

Reply:
Seems this was not even required for the next actions, so I removed it completely for this workflow.